do/csi/driver/driver_test.go

```diff
@@ -16,14 +16,23 @@ func TestDriverSuite(t *testing.T) {
 		t.Skip("skipping test suite. enable by adding the flag '-suite'")
 	}
 
+	token := os.Getenv("DIGITALOCEAN_ACCESS_TOKEN")
+	if token == "" {
+		t.Skip("skipping test suite. DIGITALOCEAN_ACCESS_TOKEN needs to be set")
+	}
+
 	socket := "/tmp/csi.sock"
 	endpoint := "unix://" + socket
 	if err := os.Remove(socket); err != nil && !os.IsNotExist(err) {
 		t.Fatalf("failed to remove unix domain socket file %s, error: %s", socket, err)
 	}
 
 	// run the driver
-	go NewDriver(endpoint).Run()
+	driver, err := NewDriver(endpoint, token)
+	if err != nil {
+		t.Fatal(err)
+	}
+	go driver.Run()
```

Comment:
Good question. I'll explose the Stop() method of the internal gRPC server. That would allow the user stop in various way, such as via `defer driver.Stop()`.